src/pages/merge/IndividualCard.jsx

```diff
@@ -53,10 +53,14 @@ export default function IndividualCard({ data, mergeConflicts }) {
           ) : (
             <Skeleton variant="text" height={30} width={140} />
           )}
-          {data && adoptionName ? (
-            <Text variant="body2">{adoptionName}</Text>
-          ) : (
-            <Skeleton variant="text" height={20} width={80} />
+          {adoptionName && (
```

Comment:
Why are adoptionName and data split into separate conditional expressions?

Reply:
the old way, if there was no adoptionName a skeleton would always show up. we only want the skeleton if there is an adoptionName that is loading